### PR TITLE
Tokyo edits

### PR DESCRIPTION
**Pull request by hadleybeeman**
Resolving issues at Tokyo f2f.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h2>Principles</h2>
...
+  <dd><p>When we are adding a feature or technology to the web, we will consider what harm it could do to society or groups, especially vulnerable people. We will prioritise potential benefits for users over potential benefits to web developers, in line with the <a href="https://www.w3.org/TR/html-design-principles/#priority-of-constituencies">priority of constituencies</a> from the HTML Design Principles. We will build new web technologies in an open and collaborative matter according to open processes (for example, the <a href="https://www.w3.org/Consortium/Process/"></a>W3C process</a>). Furthermore, by adhereing to codes of conduct (such as the W3C <a href="https://www.w3.org/Consortium/cepc/"></a>code of ethics and professional conduct</a>) we will ensure the requirements and views of marganlized communities are respected.</p></dd>
```
Comment (danbri) [inline index.html]: "an open and collaborative matter according to open processes" - this reads awkwardly. Are both "opens" needed? 

e.g. "We will build new web technologies in a collaborative matter according to open processes"

Comment (danbri) [inline index.html]: marginalized

Comment (danbri) [inline index.html]: adhering

File: index.html [hunk 2]
```diff
@@ -128,7 +128,9 @@ <h2>Principles</h2>
   <dd><p>We have a responsibility to build web technologies to counter misinformation, allowing information sources to be traceable and facts to be checkable. The concept of origin and source is core to the web's security model. We will make sure the new web technologies we create do not work against this architectural principle.</p></dd>
 
   <dt>The web must enhance individuals' control and power</dt>
-  <dd><p>The web should empower the people using it, ahead of empowering service and software developers. We will create web technologies and platforms that can't manipulate our users, complicate isolation or encourage addictive behaviors.</p></dd>
+  <dd><p> The web should empower the people using it, ahead of empowering service and software developers. We will create web technologies and platforms th\
```
Comment (danbri) [inline index.html]: > "We will create web technologies and platforms that can't manipulate our users, complicate isolation or encourage addictive behaviors"

This is a gigantic and implausible promise, as currently phrased. Any information-bearing system designed for human use can be used to manipulate, isolate and addict. Presumably the intent here is something around not making things worse, and to consider negative impacts of technology design, given that the Web is already out there and unlikely to be rescinded?

File: index.html (hunk 1 above)
Comment (koalie) [reply index.html]: @torgo instead of "matter", did you mean "manner"?

Comment (koalie) [reply index.html]: I note the inconsistent use of en-gb and en-us at least in this section. e.g. "prioritise" / "marginalized"

**Review by azaroth42 — APPROVED** [review]
Looks good to me with respect to #4, thanks all!

File: index.html (hunk 1 above)
Comment (torgo) [reply index.html]: Yes. Will fix.

Comment (torgo) [reply index.html]: fixed

Comment (torgo) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (torgo) [reply index.html]: Hi @danbri. How about the following which maybe more correctly captures the sentiment we were going for:  "We recognize that web technologies can be used by service and application developers to manipulate users, complicate isolation and encourage addictive behaviors. We recognize these risks and seek to mitigate against them when creating these technologies and platforms." ?

**Comment by torgo** [comment]
Discussed in breakout on 17-Oct. @hadleybeeman and I agreed we just need to resolve these 2 remaining issues and then we should be ready to ship an update.

File: index.html [hunk 3]
```diff
@@ -110,13 +113,13 @@ <h2>Principles</h2>
...
+  <dd><p>When we are adding a feature or technology to the web, we will consider what harm it could do to society or groups, especially vulnerable people. We will prioritize potential benefits for users over potential benefits to web developers, in line with the <a href="https://www.w3.org/TR/html-design-principles/#priority-of-constituencies">priority of constituencies</a> from the HTML Design Principles. We will build new web technologies in a collaborative matter according to open processes (for example, the <a href="https://www.w3.org/Consortium/Process/"></a>W3C process</a>). Furthermore, by adhering to codes of conduct (such as the W3C <a href="https://www.w3.org/Consortium/cepc/"></a>code of ethics and professional conduct</a>) we will ensure the requirements and views of marginanlized communities are respected.</p></dd>
```
Comment (triblondon) [inline index.html]: The new link on this line doesn't capture any text

**Review by triblondon — COMMENTED** [review]
By my understanding, this edit adds the references I asked for regarding 'open' and 'decentralised'. I think the new mentions are fairly subtle, and could be more explicit, since the terms are used without defining them.  That said, the context in which they are used implies a definition (which is particularly important for 'open'), so maybe that's OK.  My interpretation of the new use of 'open' is that it refers to transparent processes.

File: index.html (hunk 2 above)
Comment (danbri) [reply index.html]: thanks, that is much better

“complicate isolation” is a slightly awkward phrase but I can’t think of an alternative

**Comment by danbri** [comment]
to @triblondon point about defining open, afaik the closest w3c gets to defining oprn orgs is an endorsement of the “openstand” 
principles, see 
https://www.w3.org/2013/09/normative-references#orgs

https://open-stand.org/about-us/principles/

**Comment by masinter** [comment]
@danbri see https://github.com/w3c/w3process/issues/325

File: index.html (hunk 2 above)
Comment (masinter) [reply index.html]: cyberbullying is a thing: https://www.stopbullying.gov/cyberbullying/what-is-it/index.html
and there are various web sites talking about negative effects of the Internet .... (search for 'cyberbullying').

Maybe an external reference would be less awkward and more useful.

**Review by torgo — APPROVED** [review]
tag consensus dec 19 f2f to merge

**Pull request merged**